tutor/resources/styles/components/notes/summary-page.scss

```diff
@@ -220,7 +252,13 @@
             width: 100%;
           }
         }
+
+        img {
+          @media screen and ( max-width: $mobile-collapse-breakpoint ){
+            width: 85%;
```

Comment:
Looks good in the content I tested, but should we make this `max-width` instead so it doesn't enlarge smaller images?

Reply:
you are right @jivey